### PR TITLE
Update Portenta H7 power consumption

### PR DESCRIPTION
**Pull request by martab1994**
## What This PR Changes
- Update Portenta H7 power consumption

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md [hunk 1]
```diff
@@ -484,16 +484,18 @@ Laboratory equipment, Computer vision
...
 
-| Symbol  | Description                              | Min  | Typ  | Max  | Unit |
-| ------- | ---------------------------------------- | ---- | ---- | ---- | ---- |
-| VINMax  | Maximum input voltage from VIN pad       | -0.3 | -    | xx   | V    |
-| VUSBMax | Maximum input voltage from USB connector | -0.3 | -    | xx   | V    |
-| PMax    | Maximum Power Consumption                | -xx  | -    | xx   | mW   |
+| Description                                                 | Min | Typ  | Max | Unit |
+|-------------------------------------------------------------|-----|------|-----|------|
+| Power consumption in deep sleep mode with 3.7V power supply |     | 0.67 |     | mA   |
+| Power consumption in busy loop with 5V power supply         |     | 2.6  |     | mAh  |
```
Comment (aliphys) [inline content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:495]:
```suggestion
| Power consumption in busy loop with 5V power supply         |     | 2.6  |     | mA  |
```

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md [hunk 2]
```diff
@@ -484,16 +484,18 @@ Laboratory equipment, Computer vision
...
+| Maximum power consumption with 5V power supply              |     | 230  |     | mA   |
```
Comment (aliphys) [inline content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:496]: According to the power tree, the PF1550 has a LDO with a 2A output.
image
With a 5V input and at 2A output current of the LDO, then the maximum consumption by the board would be at least 2000mA. 
Could we specify what is meant by maximum in this context?

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md [hunk 3]
```diff
@@ -484,16 +484,18 @@ Laboratory equipment, Computer vision
...
+
+**Note:** All the measurements have been performed with Portenta H7 full version (ABX00042). The busy loop taken as reference consists of 595 seconds of deep sleep time and 5 seconds of power on and radio on.
```
Comment (aliphys) [inline content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:498]: Is the 2.6mA mentioned an average over the 5 seconds or across the whole 595+5 seconds?

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md [hunk 4]
```diff
@@ -484,16 +484,18 @@ Laboratory equipment, Computer vision
...
 ### Power Consumption
 
-| Symbol  | Description                              | Min  | Typ  | Max  | Unit |
-| ------- | ---------------------------------------- | ---- | ---- | ---- | ---- |
-| VINMax  | Maximum input voltage from VIN pad       | -0.3 | -    | xx   | V    |
-| VUSBMax | Maximum input voltage from USB connector | -0.3 | -    | xx   | V    |
-| PMax    | Maximum Power Consumption                | -xx  | -    | xx   | mW   |
+| Description                                                 | Min | Typ  | Max | Unit |
+|-------------------------------------------------------------|-----|------|-----|------|
+| Power consumption in deep sleep mode with 3.7V power supply |     | 0.67 |     | mA   |
```
Comment (aliphys) [inline content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:494]:
```suggestion
| Current consumption in deep sleep mode with 3.7V power supply |     | 0.67 |     | mA   |
```

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md (hunk 1 above)
Comment (aliphys) [reply content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:495]:
```suggestion
| Current consumption in busy loop with 5V power supply         |     | 2.6  |     | mAh  |
```

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md (hunk 2 above)
Comment (aliphys) [inline content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:496]:
```suggestion
| Maximum current consumption with 5V power supply              |     | 230  |     | mA   |
```

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md (hunk 3 above)
Comment (martab1994) [reply content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md]: I added a line in the note, let me know if now is more clear, thanks!

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md (hunk 2 above)
Comment (martab1994) [reply content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md]: Explained better

File: content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md [hunk 5]
```diff
@@ -484,16 +484,22 @@ Laboratory equipment, Computer vision
...
 
-| Symbol  | Description                              | Min  | Typ  | Max  | Unit |
-| ------- | ---------------------------------------- | ---- | ---- | ---- | ---- |
-| VINMax  | Maximum input voltage from VIN pad       | -0.3 | -    | xx   | V    |
-| VUSBMax | Maximum input voltage from USB connector | -0.3 | -    | xx   | V    |
-| PMax    | Maximum Power Consumption                | -xx  | -    | xx   | mW   |
+| Description                                                    | Min | Typ  | Max | Unit |
+|----------------------------------------------------------------|-----|------|-----|------|
+| Current consumption in deep sleep mode (@3.7V)                 |     | 0.67 |     | mA   |
+| Current consumption in busy loop (@5V)                         |     | 2.6  |     | mAh  |
```
Comment (aliphys) [inline content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md:495]:
```suggestion
| Current consumption in busy loop (@5V)                         |     | 2.6  |     | mA   |
```

**Review by aliphys — APPROVED** [review]
Will approve for now

**Pull request merged**